### PR TITLE
add automatic test

### PR DESCRIPTION
**Pull request by SujieZhu**
## What changes were proposed in this pull request?

add automatic test for inception_resnet_v2. Call the python inception_resnet_v2.py and freeze_graph.py code in the new folder tf/loadTest from the test code in scala.
Also check whether export the PYTHONPATH for the tensorflow models

## How was this patch tested?
unit test

### CODE REVIEW

File: spark/dl/src/test/scala/com/intel/analytics/bigdl/utils/TensorflowLoaderSpec.scala [hunk 1]
```diff
@@ -533,11 +534,69 @@ class TensorflowLoaderSpec extends FlatSpec with Matchers with BeforeAndAfter {
...
   }
+
+  private def testScriptsPath(script: String) : String = {
+    val resource = getClass().getClassLoader().getResource("tf")
+    processPath(resource.getPath()) + JFile.separator + "loadTest" +
+      JFile.separator + script
+  }
+  // test whether have export the pythonpath for the tensorflow model
+  private def tfCheck(): Unit = {
+    val exitValue = ((Seq("bash", "-c", "echo $PYTHONPATH"))!!)
```
Comment (yiheng) [inline spark/dl/src/test/scala/com/intel/analytics/bigdl/utils/TensorflowLoaderSpec.scala]: check python, tensorflow if installed

File: spark/dl/src/test/resources/tf/loadTest/freeze_graph.py [hunk 2]
```diff
@@ -0,0 +1,218 @@
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
```
Comment (yiheng) [inline spark/dl/src/test/resources/tf/loadTest/freeze_graph.py]: header.

Can we write a freeze_graph method and call it in our test scripts?

**Comment by SujieZhu** [comment]
Modify the check path and python file.

File: spark/dl/src/test/resources/tf/loadTest/inception_resnet_v2.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+    input_graph = dir + "/model/inception_resnet_v2.pbtxt"    
+    input_checkpoint = dir + "/model/inception_resnet_v2.chkp"
+    output_node_names= "InceptionResnetV2/AuxLogits/Logits/BiasAdd,InceptionResnetV2/Logits/Logits/BiasAdd,output1,output2"
+    output_graph = dir + "/inception_resnet_v2_save.pb"
+    # default value do not need to change for different models
+    restore_op_name = "save/restore_all"
+    filename_tensor_name = "save/Const:0"
+
+    input_graph_def = graph_pb2.GraphDef()
```
Comment (yiheng) [inline spark/dl/src/test/resources/tf/loadTest/inception_resnet_v2.py]: can you wrap this piece of code as a routine and use it in all module test files?

File: spark/dl/src/test/scala/com/intel/analytics/bigdl/utils/TensorflowLoaderSpec.scala [hunk 4]
```diff
@@ -533,11 +539,27 @@ class TensorflowLoaderSpec extends FlatSpec with Matchers with BeforeAndAfter {
...
+  private def testScriptsPath(script: String) : String = {
+    val resource = getClass().getClassLoader().getResource("tf")
+    processPath(resource.getPath()) + JFile.separator + "loadTest" +
+      JFile.separator + script
+  }
+
+  private def tfCheck(): Unit = {
+    val exitValue = ((Seq("python", "-c", "import sys; print ','.join(sys.path)"))!!)
+
+    if ( ! exitValue.contains("models") || !exitValue.contains("python") ) {
```
Comment (yiheng) [inline spark/dl/src/test/scala/com/intel/analytics/bigdl/utils/TensorflowLoaderSpec.scala]: why not check tensorflow?

**Comment by SujieZhu** [comment]
change to merge_checkpoint.py file and also check the tensorflow installation

**Pull request closed without merging**